### ISSUE TITLE
Collapse twitter embeds

### ISSUE THREAD

**Issue by Lunariz** [Roam-Research/issues#185]
**Describe the problem**

Twitter embeds can take up a lot of screen space. It's great that they exist, but just like sub-bullets, I'd like to be able to hide them.

**Feature implementation**

Just like bullets that have sub-bullets, there should be a triangle on the left that allows you to collapse the item. Except in this case you are collapsing this sole bullet, disabling the embed.

**Additional context**

Maybe this can be generalized for every feature. Collapse images, tables, queries?
Currently it's possible to just add an empty parent bullet and collapse that, but it looks bad.

**Comment by Lunariz** [comment]
Would be great if the collapse state for these was saved, too!
Currently there is a little collapse button, but leaving/coming back to the page uncollapses the tweet

**Comment by Nateliason** [comment]
Yeah it would be great if these were collapsed by default. It's annoying to have to hyperlink some text if I don't want the preview to show up when I don't have to do that for any other links in Roam.
